util/dvsim/Testplan.py

```diff
@@ -221,7 +224,8 @@ def map_test_results(self, test_results):
         tests_mapped = [tr.name for tr in self.test_results]
```

Comment:
Thanks Matute, i got some syntax error when changing this to a list. i will try fixing the syntax error.

Reply:
notes from discussion with Matute -> a Set is a hashtable, so in line 226 when we search for an item within the set, the search time is O(1) instead of O(N) for List :)

Reply:
https://github.com/lowRISC/opentitan/issues/13903 document here to investigate potential issues.